src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -678,6 +681,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\TargetParameterCountException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\TypeAttributes.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\TypeDelegator.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\TypeDelegator.FunctionPointer.cs" Condition="'$(FeatureNativeAot)' != 'true'" />
```

Comment:
I do not think you want this to be excluded for NativeAOT. You will get complains from the API compat validation if you do that. The APIs should be present in NativeAOT, with a temporary stubbed out implementation that just throws.

Reply:
Sure thanks; this was indented to be temporary until CoreClr was 100%.